### DISCUSSION TITLE
Is there a way to disable the comfort and nourishment effects

### DISCUSSION THREAD

**Discussion by alexnmt** [vectorwing/FarmersDelight#598] [Q&A]
I'd like to disable the nourishment and comfort effects on all foods, is there any way to disable them in the mod? I've searched the config. If not, it might be a good idea to add that as a config option in the future.

**Comment by RaymondBlaze** [comment]
Food effects are basically not configurable because they're built and defined before the comfig loads. The same applies to the effects themselves.
Despite these technical reasons, FD add-ons are also using these effects, there's no way to get all those add-ons to remove the effects.
